drivers/NFSSR.py

```diff
@@ -110,7 +110,10 @@ def validate_remotepath(self, scan):
 
     def check_server(self):
         try:
-            nfs.check_server_tcp(self.remoteserver, self.nfsversion)
+            if self.srcmd.params.get('device_config').has_key(PROBEVERSION):
```

Comment:
Can we check self.dconf? That should already be populated with all device_config parameters.